src/main/java/org/cryptimeleon/craco/sig/sps/kpw15/SPSKPW15SignatureScheme.java

```diff
@@ -36,10 +38,14 @@
 
     /**
      * The public parameters used by the scheme
-     * */
+     */
     @Represented
     SPSKPW15PublicParameters pp;
 
+    /**
+     * k is relevant in regard to the security assumptions under which the scheme is CMA-secure.
+     * See [1] (p. 12) for details.
+     */
```

Comment:
Can you add more details about how relevant it is. What is the relation and why is it in the code as a comment?

Reply:
hmm, i mean i didn't really consider that when i wrote it. I added the notes as suggested for now.
Will fix the notation later if time permits.